tests/test_timeseries_analysis.py

```diff
@@ -345,6 +345,7 @@ def test_verbosity_low_different_subsection_verbosities():
         sampling_rate=1,
         stft_window_size=2,
         verbosity_rolling_statistics=Verbosity.MEDIUM,
+        verbosity_fourier_transform=Verbosity.MEDIUM,
```

Comment:
Nitpick: This should technically be a part of a separate `test` commit, but I am OK with leaving it here since it's a very small with minimal impact.

Reply:
Yes, in this case it might be slightly better because we would be 100% sure that the old tests still pass after the refactor.
But yeah, it's a small change so I would change it. For next time I will split it.

However, in general I think that even `refactor`, `fix` or `feat` commits could contain changes in tests. This commit is borderline though.

Reply:
Yup, `refactor`, `fix`, `feat` can definitely change tests if the change in tests is required for the code change being implemented, e.g. if you re-name something in a `refactor`, of course you also rename it in tests.